homeassistant/components/foscam/config_flow.py

```diff
@@ -81,6 +111,9 @@ async def async_step_user(self, user_input=None):
             except InvalidAuth:
                 errors["base"] = "invalid_auth"
 
+            except InvalidResponse:
+                errors["base"] = "unknown"
```

Comment:
Is this really unknown? At least we know that it's an invalid camera response.

Reply:
Solved, created specific invalid response error message.